subsys/usb/class/Kconfig

```diff
@@ -15,13 +15,29 @@ config USB_CDC_ACM
 	help
 	  USB CDC ACM device class driver
 
-config CDC_ACM_PORT_NAME
+config CDC_ACM_PORT_NAME_0
 	string "CDC ACM class device driver port name"
 	depends on USB_CDC_ACM
-	default "CDC_ACM"
+	default "CDC_ACM_0"
 	help
 	  Port name through which CDC ACM class device driver is accessed
 
+if USB_COMPOSITE_DEVICE
+
+config CDC_ACM_PORT_1
+	bool "Enable virtual port 1"
+	help
+	  Enable CDC ACM Virtual port 1
```

Comment:
It is not virtual, where does it come from? Also "class device driver port name" sounds strange, why not "CDC ACM class name" or "CDC ACM interface name"?

Reply:
@jfischer-phytec-iot `interface` and `class` might be misunderstood in USB. What about 'CDC_ACM_DEVICE_1`
Also another question: Is this OK to have number of instances defined in menuconfig?

Reply:
Just wondering if we specify only number of instances, can we create number of devices with macros in `include/misc/util.h`